sympy/codegen/tests/test_ast.py

```diff
@@ -109,6 +97,29 @@ def test_AugAssign():
     raises(TypeError, lambda: aug_assign(A + A, '+', mat))
     raises(TypeError, lambda: aug_assign(B, '+', 0))
 
+
+def test_Assignment_printing():
```

Comment:
@asmeurer Here are the new tests for `repr(Assignment)`, which follow the pattern of other AST classes. The old class inherited from `Relational` and so was printed like `x := y + 1`. The new class just inherits the string representation from the base AST class.